### ISSUE TITLE
Implicit currency?

### ISSUE THREAD

**Issue by buildreactive** [lambdista/money#6]
What do you think about having an implicit currency?

That way you could do:
```
val x: Money = 1
val z = Money(1.00)
```

**Comment by lambdista** [comment]
Thank you for the suggestion but I'm not a fan of implicit conversions. I think they are confusing unless they are obvious. In this case a European will think the implicit currency is EUR while an American will think it's USD.

**Comment by buildreactive** [comment]
No, I get that... I mean, make it an option to provide an implicit -- so I would need to declare, for instance, `implicit val myLocalCurrency = EUR` or something like that?

**Comment by lambdista** [comment]
I don't know, I'd rather require explicit currency. Anyway I'll consider this, thank you.
